webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -3,6 +3,14 @@
   Protected data: <a href="{{ url('documentation:protected_data_access') }}">Protected data policy.</a>
 {%- endmacro %}
 
+{% macro source_link(link, file, line) -%}
+{% if link %}
+  <a href="{{ link }}">{{ file }}{% if line is not none %}:{{ line }}{% endif %}</a>
```

Comment:
"is not none" is the Jinja built-in test for whether the value is a Python `None`.